src/components/JobWizard/JobWizardSteps/JobWizardResources/JobWizardResources.js

```diff
@@ -84,6 +88,7 @@ const JobWizardResources = ({ formState, frontendSpec, stepIsActive }) => {
           exitEditModeTriggerItem={stepIsActive}
           fieldsPath={`${RESOURCES_STEP}.volumesTable`}
           formState={formState}
+          id="add-volumes-form-table"
```

Comment:
The same as DataInputs table
https://github.com/mlrun/ui/pull/2105/files#diff-d3cad1895d38a31464eca651332455014e5c9bd08d99197c7b0b6fb55944897dR38